examples/python/rrt-star/main.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/env python3
+"""
+Visualizes the path finding algorithm RRT* in a simple environment.
```

Comment:
I'm new to this, what is RRT* and how does it work :)
We should add a link with more learning material here and in the readme

Reply:
I have added a brief description of the algorithm in a `rr.TextDocument` with links to relevant entity paths. Here is the screenshot: 
![rrt-star-screenshot](https://github.com/rerun-io/rerun/assets/28707703/e5b3f1af-5267-4148-8873-f27649780b74)